ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -65,7 +65,7 @@ @interface EXAppViewController ()
   <EXReactAppManagerUIDelegate, EXAppLoaderDelegate, EXErrorViewDelegate, EXAppLoadingCancelViewDelegate>
 
 @property (nonatomic, assign) BOOL isLoading;
-@property (nonatomic, assign) BOOL isBridgeAlreadyLoading;
+@property (atomic, assign) BOOL isBridgeAlreadyLoading;
```

Comment:
was this left in intentionally?

Reply:
Yes that actually leads to fewer other types of crashes on spamming reload. That bool is being used effectively as a semaphore, so it should be atomic.